src/openfga.ne

```diff
@@ -0,0 +1,98 @@
+@preprocessor typescript
+
+types           -> (_newline):* (type (relations (define):+):*):* {%
+    data => data[1].map((datum: any) => {
```

Comment:
can we drop the `:any`s so that we can copy paste this in https://omrelli.ug/nearley-playground/?